frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-system-dashboard.tsx

```diff
@@ -0,0 +1,201 @@
+// Disabling as default imports from multiple files have same names
+/* eslint-disable import/no-named-default */
+/**
+ * Dashboard that is injected to ODF Extension Point
+ * TODO(bipuladh) Add this to ODF Extension Point once it's ready
+ */
+
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import {
+  DashboardsPageProps,
+  mapStateToProps,
+} from '@console/internal/components/dashboard/dashboards-page/dashboards';
+import { Page, HorizontalNav, LoadingBox, PageHeading } from '@console/internal/components/utils';
+import DashboardGrid, {
+  GridDashboardCard,
+} from '@console/shared/src/components/dashboard/DashboardGrid';
+import Dashboard from '@console/shared/src/components/dashboard/Dashboard';
+import { useFlag } from '@console/shared/src/hooks/flag';
+import { default as StatusCard } from './persistent-internal/status-card/status-card';
+import RawCapacityCard from './persistent-internal/raw-capacity-card/raw-capacity-card';
+import BreakdownCard from './persistent-internal/capacity-breakdown-card/capacity-breakdown-card';
+import UtilizationCard from './persistent-internal/utilization-card/utilization-card';
+import { default as ActivityCard } from './persistent-internal/activity-card/activity-card';
+import DetailsCard from './persistent-internal/details-card';
+import InventoryCard from './persistent-internal/inventory-card';
+import storageEfficiencyCard from './persistent-internal/storage-efficiency-card/storage-efficiency-card';
+import { StatusCard as ExtStatusCard } from './persistent-external/status-card';
+import { default as ExtBreakdownCard } from './persistent-external/breakdown-card';
+import { default as ObjectStatusCard } from './object-service/status-card/status-card';
+import { default as ExtUtilizationCard } from './persistent-external/utilization-card';
+import { DetailsCard as ObjectDetailsCard } from './object-service/details-card/details-card';
+import StorageEfficiencyCard from './object-service/storage-efficiency-card/storage-efficiency-card';
+import { BucketsCard } from './object-service/buckets-card/buckets-card';
+import { default as ObjectBreakdownCard } from './object-service/capacity-breakdown/capacity-breakdown-card';
+import DataConsumptionCard from './object-service/data-consumption-card/data-consumption-card';
+import { default as ObjectActivityCard } from './object-service/activity-card/activity-card';
+import { ResourceProvidersCard } from './object-service/resource-providers-card/resource-providers-card';
+import { OCS_INDEPENDENT_FLAG, MCG_FLAG, CEPH_FLAG } from '../../features';
+
+const convertToCard = (Card: React.ComponentType): GridDashboardCard => ({ Card });
+
+const isPagePresent = (pages: Page[], page: Page): boolean =>
+  pages.some((p) => page.href === p.href);
+
```

Comment:
Add a comment on top of this for how the sorting works. Would be useful for future reads